src/AWS.Deploy.Shell/ICommandRunner.cs

```diff
@@ -42,7 +47,7 @@ public interface ICommandLineWrapper
         /// </remarks>
         /// </param>
         /// <param name="cancelToken">
-        /// <see cref="CancellationToken"/>
+        /// <see cref="ICommandRunnerDelegate"/>
         /// </param>
         public Task Run(
```

Comment:
Can you also remove this "public" since you're in this class?

Reply:
Fixed.